### DISCUSSION TITLE
System Freeze Debian 12

### DISCUSSION THREAD

**Discussion by nemumaru** [rustdesk/rustdesk#5581] [General]
### Bug Description

When having an active connection from my Win10 PC to my Debian12 laptop, the laptop will freeze with high CPU usage and is unusable until hard reset.
Both systems run both the most recent RustDesk version and the most recent respective OS version.

### How to Reproduce

Install RustDesk on a Windows and a Debian machine. Connect from Win to Deb, do whatever task, even idle may lead to the behavior.  If you do not have an active connection to the Deb-system, it will not freeze.

### Expected Behavior

Connecting crossplatform from Windows to Debian and have both systems run as they are supposed to.

### Operating system(s) on local side and remote side

Windows 10 -> Debian 12

### RustDesk Version(s) on local side and remote side

1.2.2

### Screenshots

![debcrash](https://github.com/rustdesk/rustdesk/assets/129535850/64babbc6-4367-4a7f-9734-bdb92957eb0d)

### Additional Context

On the screenshot you see the Deb-system that just froze. In the lower right you see the colorful bars which resemble the usage of the individual CPU cores. In this case 2/4 cores are on max usage, sometimes even all four cores max out with the system then freezing. The maxed window shows "Mission Center" which does not indicate a high CPU Usage though.

**Comment by rustdesk** [comment]
try out vp8 codec from menubar
